vcd/resource_vcd_org_vdc.go

```diff
@@ -970,8 +1011,8 @@ func getVcdVdcInput(d *schema.ResourceData, vcdClient *VCDClient) (*types.VdcCon
 	}
 	computeCapacity := computeCapacityList[0].(map[string]interface{})
 
-	vdcStorageProfilesConfigurations := d.Get("storage_profile").([]interface{})
-	if len(vdcStorageProfilesConfigurations) == 0 {
+	vdcStorageProfilesConfigurations := d.Get("storage_profile").(*schema.Set)
```

Comment:
Can this return nil?

Reply:
if key doesn't have value, then default empty valued returned. Nil is returned of not existing key requested

Reply:
As far as I see `storage_profile` is mandatory so it should be safe